Feature/bridge nevm rollux design

LucasCodr opened the pull request:
# Description

This is the new design for the NEVM → Rollux bridge. The functional changes were minimal and made only to adapt to the new design, but the overall functionality remains the same as before, according to what was previously built. @eskrano, please review this pull request to verify that everything is correct.

- Next.js was updated to a newer version where it now contains a feature to add any google font with [next/font](https://nextjs.org/docs/api-reference/next/font)
- Chakra UI was added to build the new UI because we are more familiar with it, and our design systems are also based on Chakra UI design tokens and specifications.

eskrano reviewed (commented):
- could we move colours values into theme variables (just to prevent a lot of hardcoded colours)
- you've added chakra-ui package its great, but we have MUI too , do you think we can keep using MUI but with styled components ? MUI more popular than chakra . BTW open to discuss . We can left both but I am a bit afraid about app build size for users then.

LucasCodr commented:
> * could we move colours values into theme variables (just to prevent a lot of hardcoded colours)
> * you've added chakra-ui package its great, but we have MUI too , do you think we can keep using MUI but with styled components ? MUI more popular than chakra . BTW open to discuss . We can left both but I am a bit afraid about app build size for users then.

- could we move colours values into theme variables (just to prevent a lot of hardcoded colours
    - done, moved some colors to the theme
- you've added chakra-ui package its great, but we have MUI too , do you think we can keep using MUI but with styled components ? MUI more popular than chakra . BTW open to discuss . We can left both but I am a bit afraid about app build size for users then.
    - I understand your concern, but I don't think it's worth doing right now. We have limited time, and I would need to become familiar with MUI to ensure that it works as intended. Perhaps we could tackle this in the future instead. Regarding the bundle size, I don't think it's something to worry about. Additionally, I removed the `ChakraProvider` from the `_app` file so that Chakra will only be used on the specific page for the new Rollux.
